packages/p2p-v2/src/features/my-profile/screens/MyProfileStats/MyProfileStats.scss

```diff
@@ -0,0 +1,68 @@
+.p2p-v2-my-profile-stats {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    margin-bottom: 2.4rem;
+    gap: 2.4rem;
+
+    @include mobile {
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    &__item {
+        display: flex;
+        flex-direction: column;
+        border-right: 1px solid #f2f3f4;
+        padding-right: 2.4rem;
+        gap: 0.4rem;
+
+        span {
+            font-size: 14px;
+            color: #999999;
+
+            button {
+                padding-left: 0;
+                background: none;
+                font-size: 14px;
+                border: none;
+                font-style: italic;
+                cursor: pointer;
+
+                &:hover {
+                    background: none;
+                    text-decoration: none;
+                }
+            }
+        }
```

Comment:
May affect button and span outside of this scope, better to prefix it with &
```suggestion
        & span {
            font-size: 14px;
            color: #999999;

            & button {
                padding-left: 0;
                background: none;
                font-size: 14px;
                border: none;
                font-style: italic;
                cursor: pointer;

                &:hover {
                    background: none;
                    text-decoration: none;
                }
            }
        }
```

Reply:
Sass automatically knows this as nested though:
https://sass-lang.com/documentation/style-rules/
![Screenshot 2024-01-08 at 1 28 39 PM](https://github.com/binary-com/deriv-app/assets/103016120/780f5530-e6cd-4b6e-9c2a-4582a8401071)